pkg/openfeature/openfeature_test.go

```diff
@@ -329,6 +330,122 @@ func TestRequirement_1_6_1(t *testing.T) {
 	}
 }
 
+func TestRequirement_EventCompliance(t *testing.T) {
+
+	// The client MUST provide a function for associating handler functions with a particular provider event type.
+	// The API and client MUST provide a function allowing the removal of event handlers.
+	t.Run("requirement_5_2_1 & requirement_5_2_1", func(t *testing.T) {
```

Comment:
5_2_1 & 5_2_1 seems like a typo?

Reply:
Good one, it should have been 5_2_1 & 5_2_2